Sources/Commands/SwiftPackageTool.swift

```diff
@@ -1942,3 +1930,14 @@ private extension Basics.Diagnostic {
         .error("missing required argument \(argument)")
     }
 }
+
+extension BuildOperation {
+    fileprivate func buildIgnoringError(subset: BuildSubset) -> Bool {
+        do {
+            try self.build(subset: subset)
+            return true
+        } catch {
+            return false
+        }
+    }
+}
```

Comment:
we can also use observabilityScope.trap instead of this, but this preserved the previous behavior exactly

Reply:
Looks like there was only one call site for this.  Do you think there will be more, or would it be better to keep this do/catch at the one callsite?

Reply:
normally you want to do something with the error, but in this case the code only seems to care about the end result.